Source/PPFSScanner.c

```diff
@@ -2,4 +2,142 @@
 
 #include "PPFSScanner.h"
 
-#include "PPFSPrerequisites.h"
\ No newline at end of file
+#include "PPFSPrerequisites.h"
+
+char* setFilePath(char *fileName,char* folderPath)
+{
+
+	int lenght =strlen(fileName) + strlen(folderPath)+1;
+
+	char *filePath = (char*) malloc(lenght);
+
+	strcpy(filePath,folderPath);
+    strcat(filePath,"/");
+	strcat(filePath,fileName);
+
+	return filePath;
+}
+struct stat getStatFile(char* filePath)
+{
+	char buf[200];
```

Comment:
Umm, do we need a buffer? We just return the file information struct, so that should be removed.